community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md

```diff
@@ -0,0 +1,152 @@
+## Description
+
+This module creates a slurm controller node via the [SchedMD/slurm-gcp]
+[slurm\_controller\_instance] and [slurm\_instance\_template] modules.
+
+More information about Slurm On GCP can be found at the
+[project's GitHub page][SchedMD/slurm-gcp] and in the
+[Slurm on Google Cloud User Guide][slurm-ug].
+
+The [user guide][slurm-ug] provides detailed instructions on customizing and
+enhancing the Slurm on GCP cluster as well as recommendations on configuring the
+controller for optimal performance at different scales.
+
+> **_WARNING:_** The variables [enable\_cleanup\_compute] and
+> [enable\_cleanup\_subscriptions], if set to true, require addition
```

Comment:
"additional"

Reply:
The links to variables don't work. I think you should use `[var.enable_cleaunp_compute](#input_enable_cleanup_compute)` or similar

Reply:
Fixed